docs/services/thorlabs_cube_motor_kinesis.rst

```diff
@@ -7,6 +7,7 @@ This service uses bits of the official vendor Python library:
 https://github.com/Thorlabs/Motion_Control_Examples/tree/main/Python
 The service also requires the installation of the Thorlabs Kinesis software:
 https://www.thorlabs.com/software_pages/viewsoftwarepage.cfm?code=Motion_Control#
+Then, to use Thorlabs cube motors with Kinesis, you need to set the THORLABS_KINESIS_DLL_PATH environment variable.

```

Comment:
Let's put the env variable name under `` ``, to make it stand out. In the doc, it would have to be written as:
```
``THORLABS_KINESIS_DLL_PATH``
```